src/components/navigation/navigation.jsx

```diff
@@ -0,0 +1,27 @@
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+import { Link } from 'inferno-router';
+
+class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loggedIn: false,
+    };
+  }
+  render() {
+    return (
+      <div className="bar">
+        <Link className="createCrawl" to="/create">Create a Crawl</Link>
+        <Link className="header" to="/">PubCrawler</Link>
+        <div className="auth">
```

Comment:
Hm.. Why? It is only links. I am currently changing "auth" to be "right" in a separate PR.